### ISSUE TITLE
feat: support native YAML in vars file

### ISSUE THREAD

**Issue by nqb** [ovh/venom#256]
Currently, YAML dictionnaries should be formated like this:
```yaml
network.name: private
network.subnet: 172.17.1.0
```
in place of pure YAML:
```yaml
network:
  name: private
  subnet: 172.17.1.0
```
It means that it's not possible to reuse easily an existing YAML file with Venom. On top of that, it requires a lot of repetition.

**Comment by fsamin** [comment]
Hello, 
Do you have a full venom file example to illustrate it ?

**Comment by nqb** [comment]
Hello @fsamin,

This one is a YAML file from my point of view:
```yaml
---
network:
  name: private
  subnet: 172.17.1.0
```

With this file, I would like to be able to use a `var` like `{{.network.name}}` in a Venom test suite.

**Comment by fsamin** [comment]
@nqb, are you talking about var file right ? as in https://github.com/ovh/venom#run-venom-with-file-var ?

**Comment by nqb** [comment]
@fsamin, yes exactly. Sorry if I'm unclear.

I want to use a YAML file which contains my Venom vars using pure YAML syntax as reference below.

**Comment by nqb** [comment]
If possible, I would like to be able to specify in this file:
- scalar : `key: value`
- lists: 
```yaml
list:
- item 1
- item 2
```
- dict:
```yaml
dict:
  key: value
```

**Comment by fsamin** [comment]
implemented in 47441c3

**Comment by nqb** [comment]
Thanks for your work @fsamin !
